src/Messenger/Message/Idoss/DossierMessage.php

```diff
@@ -90,7 +90,9 @@ public function __construct(Affectation $affectation)
             'telephoneProprietaire' => $affectation->getSignalement()->getTelProprioDecoded(),
             'mailProprietaire' => $affectation->getSignalement()->getMailProprio(),
         ];
-        $this->descriptionProblemes = mb_strimwidth($affectation->getSignalement()->getDetails(), 0, self::DESCRIPTION_MAX_LENGTH);
+        if ($affectation->getSignalement()->getDetails()) {
+            $this->descriptionProblemes = mb_strimwidth($affectation->getSignalement()->getDetails(), 0, self::DESCRIPTION_MAX_LENGTH);
+        }
```

Comment:
`descriptionProblemes` n'a pas l'air nullable
`
$this->descriptionProblemes = $affectation->getSignalement()->getDetails() ? mb_strimwidth($affectation->getSignalement()->getDetails(), 0, self::DESCRIPTION_MAX_LENGTH) : '';
`

Reply:
??

Reply:
ok, repris !